### DISCUSSION TITLE
Alguém sabe por que a certificação GitHub Foundations não está totalmente gratuita para quem tem o acesso ao GitHub Studente Developer Pack?

### DISCUSSION THREAD

**Discussion by brennoeduardo** [community/community#143522] [GitHub Education] [answered]
### Hi everyone,

![image](https://github.com/user-attachments/assets/4f84c22f-706c-4552-97ce-c9f5267e9280)

**Comment by davevad93** [comment, marked as answer]
Hi @brennoeduardo ,

There was a bug and they had to disable the offer, technically they should have fixed it by October 31 but they haven't done it yet.  

I suggest you to subscribe to the discussion #138834 so you can stay tuned, as soon as the free certification is available again they will announce it there.

**Reply by Gabriel-Fernando-Silva** [reply]
Cara eu estou tendo este problema ainda, você saber me auxiliar como eu posso tentar pegar a certificação?? Ou o bug continua ainda?
